language/move-prover/boogie-backend-v2/src/prelude/vector-smt-array-theory.bpl

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Boogie model for vectors, based on smt arrays.
+
+// This version of vectors requires boogie to be called with `-useArrayTheory`.
+// It is not extensional.
+
+// Currently we just include the basic vector array theory.
+
+{{> vector-array-theory }}
```

Comment:
What does this mean?

Reply:
Ah, I see.  This just includes another file (which is being included in more than one place, hence the indirection).